cime_config/testdefs/testmods_dirs/clm/mpasa3p75/user_nl_clm

```diff
@@ -0,0 +1,6 @@
+! Settings currently required to run at the mpasa3p75 grid
+! urbantv files at that resolution and use a redistribution mapping
+
+stream_fldfilename_urbantv = '/glade/derecho/scratch/bdobbins/ko/tbuildmax.nc'
+stream_meshfile_urbantv = '/glade/derecho/scratch/bdobbins/ko/mesh.nc'
```

Comment:
The thing I see that I need to do here is to move these files to inputdata directory on Derecho. We plan to leave them there and NOT rimport them. As the process should work with the default resolution files. But, that requires figuring out why that doesn't work and getting a fix for it.